app/views/channels.html

```diff
@@ -14,32 +14,55 @@
         <alert ng-repeat="alert in alerts.top" type="alert.type">{{alert.msg}}</alert>
         <!-- Top Alerts -->
 
+
+
+        <ul class="list logList">
+          <li ng-repeat="entry in sortingLog track by $index" class="logItem">
+            {{entry}}
+          </li>
+        </ul>
+
         <div class="panel-body">
           <div id="no-more-tables">
-            <table class="table table-striped">
+            <table class="table table-striped channelListingTable">
               <thead>
                   <tr>
                     <th width="50">#</th>
                     <th>Channel Name</th>
-                    <th>Channel Description</th>
                     <th>Channel URL</th>
+                    <th>
+                      Priority
+                      <i ng-show="uiSettings.showTooltips" tooltip-html-unsafe="Lower Value = Higher Priority<br />Higher Value = Lower Priority" class="glyphicon glyphicon-question-sign" style="font-size: 12px"></i>
```

Comment:
Let's replace the tooltip with the following text:
> If a transaction gets matched to two or more channels, then it will get routed to the channel with the highest priority. A value of 1 is the highest possible priority and larger numbers indicate a lower priority.